src/components/search/Sidebar.jsx

```
@@ -1,28 +1,29 @@
-  PopoverContent,
-  PopoverTrigger,
-} from "@/components/ui/popover";
+import { signOut, getAuth } from "firebase/auth";
+import { Popover, PopoverContent, PopoverTrigger } from "@/components/ui/popover";
 import Playlist from "../playlist/Playlists";
-import { Link, useNavigate } from "react-router-dom";
+import { app } from "../../Auth/firebase";
+import { Link, useNavigate, useLocation } from "react-router-dom";
+import { cn } from "@/lib/utils";
+import { Separator } from "../ui/separator";
+import { ThemeToggle } from "../ThemeToggle";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ThemeToggle'. Consider to delete it or rename it.

'ThemeToggle' is defined here:


(lint/suspicious/noRedeclare)